srcpkgs/cross-x86_64-w64-mingw32/template

```diff
@@ -0,0 +1,248 @@
+_binutils_version=2.29.1
+_gcc_version=7.3.0
+_gmp_version=6.1.2
+_mpfr_version=4.0.0
+_mpc_version=1.1.0
+_isl_version=0.16.1
+_mingw_version=5.0.4
+
+pkgname=cross-x86_64-w64-mingw32
+version=${_mingw_version}
+revision=1
+short_desc="Cross toolchain for Win64"
+maintainer="Aleksey Tulinov <aleksey.tulinov@gmail.com>"
+homepage="https://sourceforge.net/projects/mingw-w64/"
+license="LGPL-3, GPL-3, ZPL"
```

Comment:
SPDX identifiers: `GPL-3.0-only, LGPL-3.0-only, ZPL-2.1`

Reply:
This is fixed in e94ff0ec, but i believe it's `LGPL-3.0-or-later`. I've aligned licenses similarly to how they are specified in other cross-compilers.